persistence/sql/identity/persister_identity.go

```diff
@@ -699,7 +699,7 @@ AND ict.name IN (?, ?)`, identifierOperator)
 	}
 
 	// Follow up: add page token support here, will be easy.
-	paginator := pop.NewPaginator(params.Page, params.PerPage)
+	paginator := pop.NewPaginator(params.Page+1, params.PerPage)
```

Comment:
Sure about this? 😅 Spent some time recently on making sure this works now properly.

Reply:
Yes, tests were failing without, and it was there previously (see diff).